frontend/file/src/main/java/com/virtuslab/gitmachete/frontend/file/MacheteFileUtils.java

```diff
@@ -21,18 +21,15 @@ public final class MacheteFileUtils {
   private MacheteFileUtils() {}
 
   public static String getSampleMacheteFileContents() {
-    // We're deliberately using \n rather than `System.lineSeparator()` here
```

Comment:
Huh TBD if this doesn't break anything on Windows

Reply:
![image](https://github.com/VirtusLab/git-machete-intellij-plugin/assets/19799111/16ab9faf-1bfa-410f-9fcc-d975db91b3ee)

Reply:
I see no errors

Reply:
Whoops sorry if I wasn't clear... pls check Color Scheme and Code Style on Windows:

<img width="665" alt="image" src="https://github.com/VirtusLab/git-machete-intellij-plugin/assets/3383210/60a3bba3-0b3a-4ebb-8703-5e1f43144a67">

Reply:
![image](https://github.com/VirtusLab/git-machete-intellij-plugin/assets/19799111/49af3dc3-6176-4030-b882-e29f0af6ebbd)

Reply:
_brak_

Reply:
![image](https://github.com/VirtusLab/git-machete-intellij-plugin/assets/19799111/9a77b489-9c9e-4bda-948f-2a716e609d4a)

Reply:
wokrs! sorry, first i checked 3.8.0